cmd/ancient-store-mem/main.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright © 2020 NAME HERE <EMAIL ADDRESS>
```

Comment:
FIXME

Reply:
I will do this.

Reply:
https://github.com/etclabscore/core-geth/pull/168/commits/2acbe19890cfdf7bfe46da32212fbf01208a9b4c